functions

```diff
@@ -152,3 +152,17 @@ service_url() {
   local SERVICE_DNS_HOSTNAME="$(service_dns_hostname "$SERVICE")"
   echo "$PLUGIN_SCHEME://postgres:$PASSWORD@$SERVICE_DNS_HOSTNAME:${PLUGIN_DATASTORE_PORTS[0]}/$DATABASE_NAME"
 }
+
+service_external_url() {
+  local SERVICE="$1"
+  local SERVICE_ROOT="$PLUGIN_DATA_ROOT/$SERVICE"
+  local PORT_FILE="$SERVICE_ROOT/PORT"
+  [[ ! -f $PORT_FILE ]] && echo '-' && return 0
+
+  local PORTS=($(cat "$PORT_FILE"))
+  local PASSWORD="$(cat "$SERVICE_ROOT/PASSWORD")"
+  local DATABASE_NAME="$(get_database_name "$SERVICE")"
+  local GLOBAL_VHOST_PATH="$DOKKU_ROOT/VHOST"
```

Comment:
What happens when this file does not exist?

Reply:
Good question, I missed it. Thanks for the point

Reply:
I did some changes here too.